pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -82,6 +82,9 @@ type ControllerOptions struct {
 	// XDSUpdater will push changes to the xDS server.
 	XDSUpdater model.XDSUpdater
 
+	// TrustDomain used as SNA
```

Comment:
What is SNA ? More comments please, the code needs to be readable by everyone, not everyone knows the details of spiffe terminology and acronyms.

Reply:
Also the terms seem to broad - should it be SpiffeTrustDomain ? Or is this a trust domain that applies to everything, including non-Spiffee forms for certs and authentication ? Either way - the comments must be detailed so people can understand.

Reply:
'SNA' was a typo (for SAN). We replaced it with 'SPIFFE identity'.